### PR TITLE
Add org.eu.encom.spectral

### PR DESCRIPTION
**Pull request by encombhat**

### CODE REVIEW

File: org.eu.encom.spectral.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+id: org.eu.encom.spectral
+runtime: org.kde.Platform
+runtime-version: 5.11
```
Comment (TingPing) [inline org.eu.encom.spectral.yaml]:
```suggestion
runtime-version: '5.11'
```

File: org.eu.encom.spectral.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=home
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
```
Comment (TingPing) [inline org.eu.encom.spectral.yaml]: Have you tested this works, I know other Qt apps have problems with it.

**Comment by TingPing** [comment]
https://encom.eu.org doesn't lead anywhere.

File: org.eu.encom.spectral.yaml (hunk 2 above)
Comment (encombhat) [reply org.eu.encom.spectral.yaml]: Yeah this works.

**Comment by encombhat** [comment]
@TingPing

**Comment by encombhat** [comment]
Is Github unstable currently? I saw your comments a minute ago but it disappeared now.

Anyway .eu.org is like .com, .org, etc.

**Comment by TingPing** [comment]
@encombhat Ok well the point of an app-id is to be unique, if you don't control the name its hard to call it unique.

**Comment by TingPing** [comment]
@MiraclePK What?

**Comment by MiraclePK** [comment]
HI,

I want to create a page on Humhub. page will include these languages like
HTML, CSS and jquery. please help


Good day

Warm Regards

Pratyush Gupta

+91 9654844685

“We can’t help everyone,

but everyone can help someone.”


[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
10/22/18,
6:13:58 AM

**Comment by encombhat** [comment]
@TingPing Well, I control encom.eu.org.

**Comment by encombhat** [comment]
And how do I need to do to prove it?

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/org.eu.encom.spectral

Thanks!

**Pull request closed without merging**